sqlmesh/core/context.py

```diff
@@ -589,8 +599,10 @@ def load(self, update_schemas: bool = True) -> GenericContext[C]:
             self.default_dialect or ""
         }
 
+        project_types = {loader.config.loader.__name__.lower() for loader in self._loaders}
+
         analytics.collector.on_project_loaded(
-            project_type=self.project_type,
+            project_type=c.HYBRID if len(project_types) > 1 else first(project_types),
```

Comment:
I don't think this is correct. The expected values are: hybrid, dbt, and native. This will return a class name instead